lib/pbench/server/database/models/tracker.py

```diff
@@ -319,6 +319,8 @@ class Dataset(Database.Base):
         # because they're terminal states that cannot be exited.
     }
 
+    ACCESS_KEYWORDS = ["public", "private"]
```

Comment:
This is certainly an improvement, but it should probably be pulling in symbolic references for the two keywords from `lib/pbench/server/api/resources/query_apis/__init__.py`....